### ISSUE TITLE
[Syncing] Label

### ISSUE THREAD

**Issue by yedhink** [bigbinary/neeto-ui#42]
## Points to note
- Component in action - `Label.js`
- Github Urls to this component:
  * [neeto-ui/Label.js](https://github.com/bigbinary/neeto-ui/tree/67ee4a5188c5feccbeca2767107651fb24932764/lib/components/Label.js)
  * [NitroUI/Label.js](https://github.com/bigbinary/neeto-help-web/tree/0b11a7b0f98ac7c533df1e9ec37cf900b4cbc901/app/javascript/Admin/Components/NitroUI/Label.js)
- :green_circle: Green color signifies how that particular line is written in `neeto-ui` package
- :red_circle: Red color signifies how that particular line is written in `NitroUI` internals

## Sync Required or Not
- Are there significant changes for `Label.js` that needs to be synced to `neeto-ui` package from `NitroUI`? - **No**
- I think using an icon over simple asterisk `*` character is an overkill

## Diff
```diff
--- NitroUI/Label.js	2020-11-28 16:54:23.984109100 +0530
+++ neetoui-official/Label.js	2020-11-28 16:49:18.737135000 +0530
@@ -16,7 +16,7 @@
   return (
     <div
       className={classnames(
-        ["flex flex-row items-center justify-between nh-label", className],
+        ["flex flex-row items-center justify-between nui-label", className],
         {
           "w-full": maxLength
         }
@@ -25,9 +25,7 @@
       <div className="flex flex-row">
         <label className="m-0">
           {children}
-          {required && (
-            <i className="ri-asterisk ri-xs align-text-bottom ml-0.5"></i>
-          )}
+          {required && "*"}
         </label>
         {helpText && (
           <Tooltip
```

**Comment by edwinbbu** [comment]
@yedhink asterisk icon is provided so that the proper margin  `ml-0.5` can be provided after the last letter. If you achieve it without icon. It's well and good.

**Comment by goutham-subramanyam** [comment]
We need to sync the changes from the KB revamp ([Ref](https://github.com/bigbinary/neeto-help-web/pull/8695/files#diff-0f3cc1118c293cbd9e12a5d5df2626d7f0b2c6df1d41b7e135fb4436cdd034e2))

**Comment by edwinbbu** [comment]
@yedhink Can you take this up next so that we can resolve issue #58 along with this.

**Comment by yedhink** [comment]
@edwinbbu Sure

**Comment by yedhink** [comment]
Fixed by #68
